automation/deploy-bin.ts

```diff
@@ -41,7 +41,11 @@ export async function createGitHubRelease() {
 			assets: finalReleaseAssets[process.platform],
 		}),
 	);
-	console.log(`Release ${version} successful: ${ghRelease.html_url}`);
+	console.log(
+		`Release ${version} successful: ${
+			(ghRelease as { html_url: any; [key: string]: any }).html_url
+		}`,
+	);
```

Comment:
Prefer to type as early as possible
```suggestion
	// const ghRelease = (await Bluebird.fromCallback(
	// ...
	)) as { html_url: any }; // (X)OR Record<string, string>, but not both, since the later one would supersede the rirst
	console.log(`Release ${version} successful: ${ghRelease.html_url}`);
```